core/src/test/java/tech/tablesaw/io/xlsx/XlsxReaderTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tech.tablesaw.io.xlsx;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tech.tablesaw.api.Table;
+import tech.tablesaw.columns.Column;
+
+public class XlsxReaderTest {
+
+    private List<Table> readN(final String name, final int expectedCount) {
+        try {
+            String fileName = name + ".xlsx";
+            final List<Table> tables = Table.read().xlsx(XlsxReadOptions.builder("../data/" + fileName));
+            Assert.assertNotNull("No tables read from " + fileName, tables);
+            Assert.assertEquals("Wrong number of tables in " + fileName, expectedCount, tables.size());
+            return tables;
+        } catch (final IOException e) {
+            fail(e.getMessage());
+        }
+        return null;
+    }
+    private Table read1(final String name, final int size, final String... columnNames) {
+        final Table table = readN(name, 1).get(0);
+        int colNum = 0;
+        for (final Column<?> column : table.columns()) {
+            Assert.assertEquals("Wrong column name", columnNames[colNum], column.name());
+            Assert.assertEquals("Wrong size for column " + columnNames[colNum], size, column.size());
+            colNum++;
+        }
+        return table;
+    }
+
+    private <T> void checkColumnValues(final Column<T> column, final T... ts) {
```

Comment:
Can you call this method `assertColumnContains` or something with the word assert in it? That will fix the Codacy warning about not having any asserts in the tests